src/Controls/tests/UITests/Tests/Issues/Issue19329.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+using System.Drawing;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue19329 : _IssuesUITest
+	{
+		public Issue19329(TestDevice device) : base(device) { }
+
+		public override string Issue => "Pointer gestures should work with relative positions correctly";
+
+		[Test]
+		public void RelativePointerPositionIsComputedCorrectly()
+		{
+			IUIElement boxView = App.FindElement("TapHere");
```

Comment:
Maybe change this to `App.WaitForElement` ? 

And that'll work better for you locally?

Reply:
My mistake :-/ I keep forgetting that Appium on WinUI can't retrieve elements that aren't visible to the accessibility tree. I changed the BoxView to a label and modified the tests slightly. 

Let me know if my modifications seem alright and then we should be good to go.